[2025-09-03T21:00:06.027+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T21:00:06.051+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.live_foreign_HNX scheduled__2025-09-03T20:59:00+00:00 [queued]>
[2025-09-03T21:00:06.061+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.live_foreign_HNX scheduled__2025-09-03T20:59:00+00:00 [queued]>
[2025-09-03T21:00:06.061+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T21:00:06.074+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): live_foreign_HNX> on 2025-09-03 20:59:00+00:00
[2025-09-03T21:00:06.082+0000] {standard_task_runner.py:63} INFO - Started process 2837085 to run task
[2025-09-03T21:00:06.086+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'live_foreign_HNX', 'scheduled__2025-09-03T20:59:00+00:00', '--job-id', '88035', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmp7g73jqj8']
[2025-09-03T21:00:06.089+0000] {standard_task_runner.py:91} INFO - Job 88035: Subtask live_foreign_HNX
[2025-09-03T21:00:06.141+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.live_foreign_HNX scheduled__2025-09-03T20:59:00+00:00 [running]> on host 1877907bc043
[2025-09-03T21:00:06.241+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='live_foreign_HNX' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T20:59:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T20:59:00+00:00'
[2025-09-03T21:00:06.242+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T21:00:06.258+0000] {foreign_HNX.py:17} INFO - Kết nối DB
[2025-09-03T21:00:06.452+0000] {foreign_HNX.py:38} INFO - Đã lưu foreign_HNX
[2025-09-03T21:00:06.453+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T21:00:06.454+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T21:00:06.466+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=live_foreign_HNX, run_id=scheduled__2025-09-03T20:59:00+00:00, execution_date=20250903T205900, start_date=20250903T210006, end_date=20250903T210006
[2025-09-03T21:00:06.499+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T21:00:06.525+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-03T21:00:06.527+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
